tests/test_csc.py

```diff
@@ -0,0 +1,154 @@
+# This file is part of dm_OCPS.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+import asynctest
+import glob
+import json
+import os
+import pathlib
+import unittest
+import yaml
+
+from lsst.ts import salobj
+from lsst.dm import OCPS
+
+STD_TIMEOUT = 2    # standard command timeout (sec)
+TEST_CONFIG_DIR = pathlib.Path(__file__).parents[1].joinpath("tests", "data", "config")
+
+
+class CscTestCase(salobj.BaseCscTestCase, asynctest.TestCase):
+    def basic_make_csc(self, initial_state, config_dir, simulation_mode,
+                       settings_to_apply='', **kwargs):
+        return OCPS.OcpsCsc(
+            initial_state=initial_state,
+            config_dir=config_dir,
+            settings_to_apply=settings_to_apply,
+            simulation_mode=simulation_mode,
+        )
+
+    async def test_default_config_dir(self):
+        async with self.make_csc(
+            initial_state=salobj.State.STANDBY,
+            config_dir=None,
+            simulation_mode=1,
+        ):
+            self.assertEqual(self.csc.summary_state, salobj.State.STANDBY)
+            await self.assert_next_summary_state(salobj.State.STANDBY)
+
+            desired_config_pkg_name = "dm_config_ocps"
+            desired_config_env_name = desired_config_pkg_name.upper() + "_DIR"
+            desired_config_pkg_dir = os.environ[desired_config_env_name]
+            desired_config_dir = pathlib.Path(desired_config_pkg_dir) / "OCPS/v1"
+            self.assertEqual(self.csc.get_config_pkg(), desired_config_pkg_name)
+            self.assertEqual(self.csc.config_dir, desired_config_dir)
```

Comment:
Somewhere (in one of these existing tests or a new test) I suggest you test the cscVersion field of the softwareVersions event (that it matches `OCPS.__version__`).